recipes/pyTEMlib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyTEMlib" %}
+{% set version = "0.2022.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 20f262cbdc89602a0897bd6324b6ed9b313496f98b2310217e9cbaadc765e444
+
+build:
+  noarch: python
+  number: 0 
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
Add the corresponding [`entry_points`](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points) item here.

Reply:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  entry_points:
    - pyTEMlib = pyTEMlib:main
```